skimage/metrics/_structural_similarity.py

```diff
@@ -252,3 +270,72 @@ def structural_similarity(im1, im2,
             return mssim, S
         else:
             return mssim
+
+
+def multiscale_structural_similarity(im1, im2,
+                          *,
+                          win_size=11,
+                          multiscale_weights=(0.0448, 0.2856, 0.3001, 0.2363, 0.1333),
+                          # \alpha, \beta values from https://ece.uwaterloo.ca/~z70wang/publications/msssim.pdf
+                          channel_axis = 2,
```

Comment:
```suggestion
                          channel_axis=None,
```
by default, we usually assume that the image is grey level :wink: